### ISSUE TITLE
Got issues connecting Python application with the mysql database

### ISSUE THREAD

**Issue by xiangshiyin** [docker/docker.github.io#12453]
File: [language/python/develop.md](https://docs.docker.com/language/python/develop/)
I was following the step-wise Python docker guide(https://docs.docker.com/language/python/develop/, section: **Use Containers for Development**), and got issues when I ran the following commands listed in the instructions
```sh
$ curl http://localhost:5000/initdb
$ curl --request POST \
  --url http://localhost:5000/widgets \
  --header 'Content-Type: application/x-www-form-urlencoded' \
  --data 'name=widget01' \
  --data 'description=this is a test widget'
```

The error message I got:
```sh
$ curl http://localhost:5000/initdb
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>
```

I was following exactly the same code samples listed on the webpage. Could anyone help take a look? Thanks!

**Comment by guillaumeklimek** [comment]
Hey @xiangshiyin,

I just found 2 issues in this part, I give a solution here https://github.com/docker/docker.github.io/issues/12518 

Regards,
Guillaume

**Comment by tymik** [comment]
Fixed by #12559
